chainercv/links/connection/conv_2d_activ.py

```diff
@@ -0,0 +1,73 @@
+import chainer
+from chainer.functions import relu
+from chainer.links import Convolution2D
+
+
+class Conv2DActiv(chainer.Chain):
+    """Convolution2D --> Activation
+
+    This is a chain that does two-dimensional convolution
+    and applies an activation.
+
+    The arguments are the same as those of
+    :class:`chainer.links.Convolution2D`
+    except for :obj:`activ`.
+
+    Example:
+
+        There are sevaral ways to initialize a :class:`Conv2DActiv`.
+
+        1. Give the first three arguments explicitly:
+
+            >>> l = Conv2DActiv(5, 10, 3)
+
+        2. Omit :obj:`in_channels` or fill it with :obj:`None`:
```

Comment:
The order of descriptions should be same as that of examples.
I mean, `Fill :obj:in_channels with :obj:None: or omit it:` is better. (Changing the order of examples is also OK)